.coverage/lcov-report/index.html

```diff
@@ -0,0 +1,80 @@
+<!doctype html>
```

Comment:
You’re absolutely correct. I’m sorry about this. The `.gitignore` didn’t pick it up because I changed the folder name and I pushed it accidentally. Thankfully this pose a security issue, so I’m gonna add the `.coverage/` folder to the `.gitignore` list so it is ignored from now on.

Thanks for catching this @abungay 🍻